src/app/routes/registration/enrolment-form/enrolment-form.component.html

```diff
@@ -1,24 +1,27 @@
 <form role="form" [formGroup]="enrolmentForm">
     <div formGroupName="registrationTypes">
         <mat-checkbox class="pl-3 md-enroll"
-                      data-qa-id="on-chain"
-                      formControlName="onChain"
-                      [ngStyle]="txtboxColor">
-                  <span class="mb-3 d-inline-flex"
-                        [ngStyle]="txtColor"
-                        matTooltip="Short description of what is On-chain.">
-                      On-chain
-                  </span>
+            data-qa-id="on-chain"
+            formControlName="onChain"
+            [ngStyle]="txtboxColor"
+        >
```

Comment:
Why are you putting closing tag in next line?

Reply:
When I use react. Eslint requires the brackets to be on the same line. So I kinda formed the habit.

Reply:
But for that, I made those changes to force a rebuild.

Reply:
please use a single standard for this project